zerver/webhooks/appveyor/view.py

```diff
@@ -1,4 +1,5 @@
 from django.http import HttpRequest, HttpResponse
+from pydantic.alias_generators import to_snake
```

Comment:
We could write our own code for this, of course, since it is typically considered better to use some library's function, I went with this.